modules/govuk_jenkins/templates/jobs/copy_data_to_integration.yaml.erb

```diff
@@ -60,6 +70,11 @@
             HOSTNAME=deploy.integration.publishing.service.gov.uk
             API_KEY=<%= @ci_alphagov_api_key %>
             AUTH_TOKEN=<%= @auth_token %>
+      - slack:
+          team-domain: <%= @slack_team_domain %>
+          auth-token: <%= @environment_variables['SECOND_LINE_SLACK_AUTH_TOKEN']%>
```

Comment:
Does this work? I thought this would need to be declared in the manifest that created the job. Still, I know there are some peculiarities between where variables are defined and used in templates so wouldn't be surprised if this does work.

If we're getting the secret by calling the hash into the job definition itself, does it also need to be set as a global env var?